content/placeholders/abstract.php

```diff
@@ -169,11 +168,6 @@ public function setAttributes($attributes)
         return $this->attributes = (array)$attributes;
     }
 
-    public function getVaryAttributes()
-    {
-        return $this->varyAttributes;
-    }
-
     public function setVaryAttributes($attributes)
```

Comment:
Remove all methods related to varyAttributes.. make sure these methods are no called anywhere even in pro plugin.